packages/platform/types/slots.ts

```diff
@@ -23,3 +32,24 @@ export class GetAvailableSlotsInput {
   @IsOptional()
   duration?: number;
 }
+
+export class RemoveSelectedSlotInput {
+  @IsString()
+  @IsOptional()
+  uid?: string;
+}
+
+export class ReserveSlotInput {
+  @IsInt()
+  eventTypeId!: number;
+
+  @IsDate()
+  slotUtcStartDate!: string;
```

Comment:
Suggesting we add `Time` onto this to follow our new naming standards 
```suggestion
  slotUtcStartDateTime!: string;
```